docs/sms-send-how-to.md

```diff
@@ -51,9 +57,12 @@ Then you can add `messaging-grpc` and `utils-grpc`:
 
 <ClientDependencies :clients="['messaging-grpc', 'utils-grpc']"/>
 
-### Send SMS
+### Send text SMS
 <GithubCode :to="$frontmatter.code" />
 
+### Send binary SMS
+<GithubCode :to="https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendBinarySms.kt" />
+
 ## Resources
 * [SendSmsDemo.kt](https://github.com/working-group-two/wgtwo-kotlin-code-snippets/blob/master/src/main/kotlin/com/wgtwo/example/sendsms/SendSmsDemo.kt)
```

Comment:
Not really part of the change, but remove this?

Reply:
Done